torch_sim/models/orb.py

```diff
@@ -382,7 +385,7 @@ def forward(self, state: SimState | StateDict) -> dict[str, torch.Tensor]:
             state = state.to(self._device)
 
         half_supercell = (
-            torch.max(state.volume) > 1000
+            torch.max(torch.det(state.cell)) > 1000
```

Comment:
this is because volume isn't defined for PBC=False

Reply:
This also won't be defined for `PBC=False` because `state.cell` will be `None`.

What about:

```
        half_supercell = (
            torch.max(state.volume) > 1000
            if state.pbc and self._half_supercell is None
            else self._half_supercell
        )
```